src/diff.ts

```diff
@@ -0,0 +1,28 @@
+import crypto from 'crypto';
+import * as core from '@actions/core';
+import { Repo } from './github';
+import { VersionResponse } from 'bump-cli';
+import { bumpDiffComment } from './common';
+
+const commentTitle = '🤖 Documentation diff detected:';
+const codeBlock = '```';
+const poweredByBump = '> _Powered by [Bump](https://bump.sh)_';
+
+export async function run(version: VersionResponse): Promise<void> {
+  if (!version.diff_lines) {
+    core.info('No diff found, nothing more to do.');
+    return;
+  }
+
+  const repo = new Repo();
+  const digest = crypto
+    .createHash('sha1')
+    .update(version.diff_lines.join('\n'), 'utf8')
+    .digest('hex');
+  const body = [commentTitle, codeBlock]
+    .concat(version.diff_lines)
+    .concat([codeBlock, poweredByBump, bumpDiffComment(version.id, digest)])
+    .join('\n');
```

Comment:
This whole part will change is we now return the diff summary directly as a string.